src/lib/Navbar.svelte

```diff
@@ -33,14 +33,22 @@
       expanded: false,
       Items: [
         {
-          name: 'Getting Started with Beat Saber',
+          name: 'Get Started with Beat Saber',
           href: '/getting-started',
         },
         {
-          name: 'Getting Started with Custom Songs',
+          name: 'Get Started with Custom Songs',
           href: '/getting-started/custom-songs',
-        }
-      ]
+        },
+        {
+          name: 'Get Started with Custom Mods',
+          href: '/installing-the-mod-guide-necessary-for-any-custom-songs',
+        },
+        {
+          name: 'Get Started with Mapping',
```

Comment:
I did forget to raise this point in the last PR:
I think the sub menu names are too long. How about something like `Get Started` for the menu and then sub menus can be just `With Beat Saber` or even shorter `Beat Saber`. It think it'd be preceived as one unit due to the hierachy that is conciously interacted with prior + the intend for the subemenu names.